### PR TITLE
Docker image, Code 128 barcodes, gridlines, LCRY-2380 template, and API doc

### PR DESCRIPTION
**Pull request by tsibley**
See the commit messages for the grab bag of features in this branch.

### CODE REVIEW

**Comment by tsibley** [comment]
I added a license as well, but you may want to choose a different one than MIT.

**Comment by silby** [comment]
- [x] I don't really want to install Docker. Does it work?
- [x] The Code 128 macro is plain TeX. I plan to add support for labels on rolls for use with a label printer sometime soon and I'm just using plain (Xe)TeX. If we update the commit message in your import I won't have to check on this next time.
- [ ] Move API documentation to its own page; template in the actual deployed host and mount path in the example.

**Comment by silby** [comment]
oh cool it looks like I had a comment disappear due to eventual consistency or something I love this product

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Copyright © 2014—2018 Mullins Lab
```
Comment (silby) [inline LICENSE]: copyright slug should just be "University of Washington"

**Comment by tsibley** [comment]
Docker works.

LaTeX vs. TeX in commit messages corrected. I'm bad at caring about the distinction.

Copyright holder updated.

API doc moved to its own page. I didn't template the current host/port because the page is currently just a static file and adding templating to it seems like overkill.

**Pull request merged**